src/sentry/api/endpoints/organization_events_v2.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
+
+from functools import partial
+
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsV2EndpointBase, OrganizationEventsError, NoProjects
+from sentry.api.paginator import GenericOffsetPaginator
+from sentry.utils.snuba import transform_aliases_and_query
+
+
+class OrganizationEventsV2Endpoint(OrganizationEventsV2EndpointBase):
+
+    def get(self, request, organization):
```

Comment:
Just a note. This probably shouldn't be tackled in this pr. But it doesn't appear like there's anything similar to direct hit from ` OrganizationEventsEndpoint`. I'm not sure what would happen there (just bring up the event modal maybe?), but I wanted to note that is a feature that is not in this endpoint as written.

Reply:
I'll add this to our backlog - I guess it makes sense to keep support for this